common/src/main/java/org/vivecraft/client_vr/gui/GuiKeyboard.java

```diff
@@ -42,8 +44,10 @@ public void init() {
                 }
 
                 String s2 = String.valueOf(c0);
+                final int code = k < this.dataholder.vrSettings.keyboardCodes.length ? this.dataholder.vrSettings.keyboardCodes[k] : GLFW.GLFW_KEY_UNKNOWN;
```

Comment:
k here should be l1, maybe would be a good idea to readd the named variables here too